lib/fs-wrapper.js

```diff
@@ -1,7 +1,10 @@
+const path = require('path');
 const util = require('util');
 const fs = require('graceful-fs');
 const {rimraf} = require('rimraf');
 
+const copyFile = util.promisify(fs.copyFile);
```

Comment:
Btw, there’s a a promises API: https://nodejs.org/api/fs.html#promises-api

Reply:
Ah yes, I looked for it in my code but didn't see any usages, so I thought they must have appeared after v10.

Reply:
From the page above:

> Version | Changes
> -- | --
> v14.0.0 | Exposed as require('fs/promises').
> v11.14.0, v10.17.0 | This API is no longer experimental.
> v10.1.0 | The API is accessible via require('fs').promises only.
> v10.0.0 | Added in: v10.0.0

So you can use it in 10+, but it'll be experimental, and getting at it might be annoying.